internal/globalMerge.ecl

```diff
@@ -138,7 +138,7 @@ EXPORT globalMerge := MODULE
             if(rc < pc)
             {
               row = rowBuilder.ensureCapacity(returnsize, NULL);
-              void * pos = row;
+              unit8_t * pos = row;
```

Comment:
typo.  I think this should be uint8_t, (transposition of n and i).  This shouldn't have compiled.